bin/jsonizer/include/nil/proof-producer/recursive_json_generator.hpp

```diff
@@ -72,27 +71,26 @@ namespace nil {
             using pow_operation_type = typename constraint_system_type::pow_operation_type;
             using assignment_table_type = typename PlaceholderParams::assignment_table_type;
 
-
-            static std::string generate_field_array2_from_64_hex_string(std::string str){
+            static std::string generate_field_array2_from_64_hex_string(std::string str) {
                 BOOST_ASSERT_MSG(str.size() == 64, "input string must be 64 hex characters long");
                 std::string first_half = str.substr(0, 32);
                 std::string second_half = str.substr(32, 32);
-                return  "{\"vector\": [{\"field\": \"0x" + first_half + "\"},{\"field\": \"0x" + second_half + "\"}]}";
+                return "{\"vector\": [{\"field\": \"0x" + first_half + "\"},{\"field\": \"0x" + second_half + "\"}]}";
             }
 
             template<typename HashType>
-            static inline std::string generate_hash(typename HashType::digest_type hashed_data){
-                if constexpr(std::is_same<HashType, nil::crypto3::hashes::sha2<256>>::value){
+            static inline std::string generate_hash(typename HashType::digest_type hashed_data) {
```

Comment:
Make this a const reference?

Reply:
I didn't touch json-related things in this PR. Not sure if it is needed and why it was moved to extra binary